app/views/devise/registrations/new.html.erb

```diff
@@ -1,39 +1,35 @@
-<h2>Sign up</h2>
-
+<div class="register">
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
-
-   <div class="field">
-    <%= f.label :name %>
+ 
+  <div class="register_fields_container"> 
+   <div class="lines">
+    <%= f.label :full_name%>
     <%= f.text_field :name, autocomplete: "Name", autofocus: true %>
   </div>
-
-   <div class="field">
-    <%= f.label :last_name %>
-    <%= f.text_field :last_name, autocomplete: "Last name", autofocus: true %>
-  </div>
-
-  <div class="field">
-    <%= f.label :email %><br />
+    
+  <div class="lines">
+    <%= f.label :email %>
     <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
   </div>
 
-  <div class="field">
+  <div class="lines">
     <%= f.label :password %>
     <% if @minimum_password_length %>
-    <em>(<%= @minimum_password_length %> characters minimum)</em>
-    <% end %><br />
+    <em>(<%= @minimum_password_length %> characters min)</em>
+    <% end %>
     <%= f.password_field :password, autocomplete: "new-password" %>
   </div>
 
-  <div class="field">
-    <%= f.label :password_confirmation %><br />
+  <div class="lines">
+    <%= f.label :password_confirmation %>
     <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
   </div>
 
-  <div class="actions">
-    <%= f.submit "Sign up" %>
+  <div class="sign_up_button_container">
+    <p><%= f.submit "Sign up", class: "sign_up_button" %></p>
   </div>
+</div>
```

Comment:
@HFG43 I kindly suggest you add custom styles for the button and give a little more space between the form fields and the button.